tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "moduleResolution": "node",
```

Comment:
`ncc` version `0.28.0` requires that for some reason.
I am a bit surprised that it is not the default. Quoting [typescript doco](https://www.typescriptlang.org/docs/handbook/module-resolution.html#module-resolution-strategies) :
> There are two possible module resolution strategies: Node and Classic. You can use the --moduleResolution flag to specify the module resolution strategy. If not specified, **the default is Node for --module commonjs**, and Classic otherwise (including when --module is set to amd, system, umd, es2015, esnext, etc.).